tests/unittests/BackendTestUtils2.h

```diff
@@ -298,6 +296,12 @@ void inferExtract3D(Tensor *input, Tensor *out, llvm::StringRef kind);
 
 void inferMaxSplat(Tensor *input, Tensor *out, llvm::StringRef kind);
 
+void insertCompiledFunction(llvm::StringRef name, CompiledFunction *func,
+                            runtime::DeviceManager *device, Module *mod);
+
+void runOnDevice(ExecutionContext &context, llvm::StringRef name,
```

Comment:
Looks like this is unused. Is it going to be used by a future PR, if so maybe add it there?

Reply:
These are both used in other PRs, I was porting BackendTestUtils as a whole, I can back these out until a later PR.